tips/technology-resources.md

```diff
@@ -222,28 +223,40 @@
 
 ## Javascript
 
-- [Arrayzing](https://gist.github.com/ourmaninamsterdam/1be9a5590c9cf4a0ab42#user-content-create-an-array) - Gives you answer on how to manage arrays by indexing topics from what you want to do, not the method names
+#### Books
+
 - [Clean Code JavaScript](https://github.com/ryanmcdermott/clean-code-javascript) - Clean Code concepts adapted for JavaScript
-- [D3 - Data-Driven Documents](https://d3js.org/) - A fantastic javascript library for manipulating documents based on data
-- [Dwitter](https://www.dwitter.net/) - JavaScript demos in 140 characters or less
-- [ECMA-262](https://tc39.github.io/ecma262/) - The defacto living standards guide for JavaScript
+- [Deep JavaScript](https://exploringjs.com/deep-js/toc.html) - A deep dive into JavaScript and ECMAScript
 - [Eloquent Javascript by Marijn Haverbeke](https://github.com/marijnh/Eloquent-JavaScript) - A great free book the learning JavaScript
 - [Exploring ES2016 and ES2017](http://exploringjs.com/es2016-es2017/) - A deep dive into changes in ES2016 and 2017
-- [Greensock Animations Library](https://greensock.com/) - A supurb animations library that is easy to implement in JavaScript and is highly efficient
 - [JavaScript Allongé](https://leanpub.com/javascriptallongesix) - A book on functional programming for JavaScript
 - [JavaScript Design Patterns](https://addyosmani.com/resources/essentialjsdesignpatterns/book/) - A thorough look at JavaScript Design Patterns
-- [JavaScript Patterns](http://shichuan.github.io/javascript-patterns/) - A list of design patters for JavaScript
+- [JavaScript Patterns](http://shichuan.github.io/javascript-patterns/) - A list of design patterns for JavaScript
 - [JavaScript Spessore](https://github.com/raganwald/javascript-spessore) - A follow up for [JavaScript Allongé](https://leanpub.com/javascriptallongesix), it dives deeper into similar concepts, not strictly about functional programming
-- [JavaScript Visualizer](https://tylermcginnis.com/javascript-visualizer/) - A JavaScript visualizer that focuses on execution context
+- [Professor Frisby’s Mostly Adequate Guide to Functional Programming](https://github.com/MostlyAdequate/mostly-adequate-guide) - A deep dive into functional programming in JavaScript
+- [You Don't Know JS by Kyle Simpson](https://github.com/getify/You-Dont-Know-JS) - A series of JavaScript books that cover from 'Getting Started' to 'ES6 and Beyond'
+
+#### Documentation/Courses
+
+- [ECMA-262](https://tc39.github.io/ecma262/) - The defacto living standards guide for JavaScript
 - [The Modern JavaScript Tutorial](https://javascript.info/) - Free tutorials covering most relevant JavaScript topics
+- [Rithm School Intermediate JavaScript](https://www.rithmschool.com/courses/intermediate-javascript) - A course moving past the basics of JavaScript|
```

Comment:
Very cool - One of my coworkers went to Rithm.

Reply:
I just found out Rithm supports Springboard Software Engineering. It's all connected man.

Reply:
:big-brain: